styles.css

```diff
@@ -293,3 +292,85 @@ h3 {
   display: flex;
   flex-direction: column;
 }
+
+.contact {
+  background-image: url('images/image_geometry_1.jpg');
+  background-position: left top;
+  background-repeat: no-repeat;
+  background-size: 111px 74px;
+  margin-top: 27px;
+  padding-inline: 18px 14px;
+  padding-block: 106px 31px;
+}
+
```

Comment:
- [ ] Since this is a group project, please consider adding all the contributed social account links to the  **README** documentation 👍🏼